lib/decimal.ex

```diff
@@ -1409,34 +1410,37 @@ defmodule Decimal do
     |> new()
   end
 
-  @doc """
-  Creates a new decimal number from an integer, string, float, or existing decimal number.
-
-  Because conversion from a floating point number is not exact, it's recommended
-  to instead use `new/1` or `from_float/1` when the argument's type is certain.
-  See `from_float/1`.
-
-  If the value cannot be cast, Decimal.Error is raised.
-
-  ## Examples
-
-      iex> Decimal.cast(3)
-      #Decimal<3>
-
-      iex> Decimal.cast(3.0)
-      #Decimal<3.0>
-
-      iex> Decimal.cast("3")
-      #Decimal<3>
-
-      iex> Decimal.cast("3.0")
-      #Decimal<3.0>
-
-      iex> Decimal.new(3) |> Decimal.cast()
-      #Decimal<3>
-
-  """
+  # TODO: uncomment on decimal 2.0
```

Comment:
Lets remove the comments since it needs to be updated with the new return values and the other docs may not be relevant.